modules/router-store/src/router_store_module.ts

```diff
@@ -122,31 +122,15 @@ enum RouterTrigger {
  *       { path: '', component: SimpleCmp },
  *       { path: 'next', component: SimpleCmp }
  *     ]),
- *     StoreRouterConnectingModule
+ *     StoreRouterConnectingModule.forRoot()
  *   ],
  *   bootstrap: [AppCmp]
  * })
  * export class AppModule {
  * }
  * ```
  */
-@NgModule({
-  providers: [
-    {
-      provide: _ROUTER_CONFIG,
-      useValue: {},
-    },
-    {
-      provide: ROUTER_CONFIG,
-      useFactory: _createRouterConfig,
-      deps: [_ROUTER_CONFIG],
-    },
-    {
-      provide: RouterStateSerializer,
-      useClass: DefaultRouterStateSerializer,
-    },
-  ],
-})
+@NgModule()
```

Comment:
It should be an empty object even if there's no metadata.

```suggestion
@NgModule({})
```

Reply:
Oh. Thanks for your advises. I hope my changes now meet the requirements.